components/automate-gateway/api/nodes/manager/manager.proto

```diff
@@ -82,6 +128,12 @@ service NodeManagerService {
 			action: "infra:nodeManagers:delete"
 		};
 	};
+
+	/*
+	Delete a nodemanager and set nodes to have a state of stopped
+
+	Delete a nodemanager and update all associated nodes to have a state of stopped. 
```

Comment:
Consider delineating keywords--such as "stopped" here--with a style; presumably, there is a convention for that.